scripts/test_data_generator/generate_iceberg.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/python3
+#!/Users/mritchie712/opt/anaconda3/bin/python
```

Comment:
this will break on other computers. In this file there are some useful changes (partitioning) but also spurious changes likes semicolons and spacing.. It would be nice to remove the spurious changes to make the diff as small as possible